lib/fetch/sanitize.js

```diff
@@ -164,22 +168,46 @@ export function sanitizeBadger(json) {
     lastName: name && name.split(' ').slice(-1).join(' '),
     order: get('badger.order'),
     jobTitle: get('badger.job-title'),
-    primaryImageUrl: primaryImageUrl && primaryImageUrl.replace(/^http:/, 'https:'),
-    secondaryImageUrl: secondaryImageUrl && secondaryImageUrl.replace(/^http:/, 'https:'),
+    primaryImageUrl: primaryImageUrl &&
```

Comment:
If you're adopting prettier - you want to do it across the whole codebase at once, in a separate PR, surely?